pontoon/projects/static/js/multiple_project_selector.js

```diff
@@ -0,0 +1,56 @@
+// Contains behaviours of widgets that are shared between admin and end-user interface.
+$(function () {
+  /**
+   * Function keeps track of inputs that contain information about the order of selected projects.
+   */
+  function updateSelectedProjects() {
+    const $selectedList = $('.multiple-project-selector .project.selected'),
+      $selectedProjectsField = $selectedList.find('input[type=hidden]'),
+      selectedProjects = $selectedList
+        .find('li[data-id]')
+        .map(function () {
+          return $(this).data('id');
+        })
+        .get();
```

Comment:
Not really sure why we persist with jQuery; in usage like this it's not really any better than the pure JS alternative:
```js
const listItems = document.querySelectorAll('.multiple-project-selector .project.selected li[data-id]');
const selectedProjects = Array.from(listItems, li => li.dataset['id']);
```

This isn't intended as a comment on just this specific place, but overall we should not be making any later transition away from jQuery progressively harder

Reply:
`multiple_project_selector` widget is reusing HTML, JS and CSS code from the `multiple_locale_selector` widget. Should I refactor that into a generic `multiple_item_selector` widget?

Reply:
I think we should have a discussion about what we're doing with the non-translate parts of the UI. I'm not really concerned about this or any other specific instance, but more the overall tech stack that we're using, and whether we might want to move towards less jQuery use.

Reply:
Filed https://github.com/mozilla/pontoon/issues/3315.